TeamCode/src/main/java/org/firstinspires/ftc/teamcode/auton/DistanceSensorTest.java

```diff
@@ -0,0 +1,24 @@
+package org.firstinspires.ftc.teamcode.auton;
+
+import com.qualcomm.robotcore.eventloop.opmode.Autonomous;
+import com.qualcomm.robotcore.eventloop.opmode.OpMode;
+import com.qualcomm.robotcore.hardware.DistanceSensor;
+
+import org.firstinspires.ftc.robotcore.external.navigation.DistanceUnit;
+
+@Autonomous(name="Distance Sensor Test")
+public class DistanceSensorTest extends OpMode {
+    DistanceSensor distanceSensor;
+    @Override
+    public void init() {
+        telemetry.addData("Initializing Distance Sensor.", null);
+        telemetry.addData("Distance Sensor Initialized.", null);
+    }
+
+    @Override
+    public void loop() {
+        distanceSensor = hardwareMap.get(DistanceSensor.class, "distanceSensor");
```

Comment:
let's write a version of this that uses the `hardwareElement` class, just to make sure that you implemented it correctly there

Reply:
can be the same file, can be a different file, up to you